src/main/java/fruit/shop/records/parser/RecordsParserCommaSeparated.java

```diff
@@ -0,0 +1,34 @@
+package fruit.shop.records.parser;
+
+import fruit.shop.strategy.ActivityStrategy;
+import fruit.shop.strategy.ActivityStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RecordsParserCommaSeparated implements RecordsParser {
+    public static final String SEPARATOR = ",";
+    public static final int FRUIT_INDEX = 1;
+    public static final int OPTION_INDEX = 0;
+    public static final int VALUE_INDEX = 2;
+    public static final int DEFAULT_VALUE = 0;
+
+    @Override
+    public Map<String, Integer> saveRecordsToMap(List<String> records) {

```

Comment:
Mehtod saveRecordsToMap parse data and use strategy for operationHandler
You should divide this logic.

```
public interface ParserService {
    List<FruitTransaction> parseData(List<String> inputData);
}
```
and

```
public interface FruitShopService {
    void processOfOperations(List<FruitTransaction> fruitTransactions);
}
```